spec/lib/healthchecks/queue_latency_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
```

Comment:
The file ought to be named `queue_latency_check_spec.rb` so it maps to directly to the Ruby source file. A lot of automated test runners use that as the logic for finding the spec test for a matching file.

Reply:
Thanks for the tip!